bundles/org.openhab.binding.sonnen/src/main/resources/OH-INF/i18n/sonnen_fr.properties

```diff
@@ -14,6 +14,9 @@ thing-type.config.sonnen.sonnenbattery.hostIP.label = Adresse IP
 thing-type.config.sonnen.sonnenbattery.hostIP.description = Veuillez ajouter l'adresse IP de votre batterie Sonnen.
 thing-type.config.sonnen.sonnenbattery.refreshInterval.label = Fréquence d'actualisation
 thing-type.config.sonnen.sonnenbattery.refreshInterval.description = Nombre de secondes entre deux actualisations programmées par la batterie Sonnen après qu'un canal ait été lié à un élément. Entrez une valeur de 0 à 1000.
+thing-type.config.sonnen.sonnenbattery.authToken.label = Jeton d'authentification
+thing-type.config.sonnen.sonnenbattery.authToken.description = Token d'authentification qui peut �tre trouv� sous "Software Integration" si vous vous connectez localement � votre batterie sonnen. S'il est vide, l'ancienne API d�pr�ci�e sera utilis�e.
+
```

Comment:
Please revert translations, which should be provided in Crowdin after the PR is merged. For reference: https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations

Reply:
done